public/src/pages/auth/Login/Login.tsx

```diff
@@ -74,9 +76,16 @@ export const Login: FC = () => {
         setLoginResponse(data);
         return data;
       })
-      .then(({ email, errorText }) => {
-        setErrorText(errorText);
+      .then(async ({ email, errorText }) => {
+        errorText ?? setErrorText(errorText);
         sessionStorage.setItem('email', email);
+        return getUserData(email);
+      })
+      .then((userData: UserData) => {
+        sessionStorage.setItem(
+          'userName',
+          `${userData.firstName} ${userData.lastName}`
+        );
       });
```

Comment:
You can remove useless brackets here

```suggestion
      .then((userData: UserData) => sessionStorage.setItem(
        'userName',
        `${userData.firstName} ${userData.lastName}`
      ));
```